account/private.go

```diff
@@ -84,47 +84,47 @@ func PrivateKeyFromBase58Seed(seedBase58Encoded string) (*PrivateKey, error) {
 	// verify length
 	seed := util.FromBase58(seedBase58Encoded)
 	seedLength := len(seed)
-	if base58EncodedSeedV1Length != seedLength && base58EncodedSeedV2Length != seedLength {
+	if SeedV1Length != seedLength && SeedV2Length != seedLength {
 		return nil, fault.ErrInvalidSeedLength
 	}
 
 	// verify checksum
 	digest := sha3.Sum256(seed[:seedLength-checksumLength])
-	checksumStart := seedLength - seedChecksumLength
-	expectedChecksum := digest[:seedChecksumLength]
+	checksumStart := seedLength - SeedChecksumLength
+	expectedChecksum := digest[:SeedChecksumLength]
 	actualChecksum := seed[checksumStart:]
 	if !bytes.Equal(expectedChecksum, actualChecksum) {
 		return nil, fault.ErrChecksumMismatch
 	}
 
 	header := seed[:seedHeaderLength]
-	var encryptedSk []byte // encrypted seed for generate key pair
-	var isTest bool        // denote the network is test net
+	var encryptedSk []byte // encrypted secret key for generate key pair
+	var testnet bool       // denote the network is test net
```

Comment:
I answer [here](https://github.com/bitmark-inc/bitmarkd/pull/61#discussion_r297479719)

Reply:
testnet is ok as flag value
however encrypted seems to be a problem since I do not think anything is encrypted

Reply:
We have different algorithm for generate ed25519 key from the "seed". The "seed" to generate ed25519 is different from our seed. I mean that our seed is the 32 byte slice (i call it as secret key) combine with header, version, network (for 24 words), checksum then encode to base58 to get the base58 seed. The "seed" that use to generate ed25519 key pair is also different in v1 and v2 (v1 need to seal by `secret_box` and v2 need to shake 4 times with sha256). So the `encryptedSk` means the "seed" that i mentioned. So could you propose the better name, @hxw?

Reply:
Since it's totally a ed25519 seed for generating the key pair, so name it as `ed25519Seed` is better.

Reply:
It is not encrypted, it is more like a master key, or a generator key.
Not the eed25519 seed because that is exactly the private key and this values goes through some scrambling process to become that key.  At one point it was named "core" but that seems confusing.
Maybe "parent" or "stem" might work